src/main/java/com/toptrumps/cli/CommandLineController.java

```diff
@@ -80,17 +81,62 @@ public void onAttributeSelected(Player activePlayer) {
 
     @Override
     public void onRoundEnd(RoundOutcome outcome) {
-        System.out.println("Round finished");
+        showRoundResult(outcome);
+        ArrayList<Player> removedPlayers = outcome.getRemovedPlayers();
+        if(!removedPlayers.isEmpty()){
+            showEliminatedPlayers(removedPlayers);
+        }
+    }
+
+    private void showRoundResult(RoundOutcome outcome){
+        RoundOutcome.Result result = outcome.getResult();
+        String roundResult = "\nThe round resulted in a: ";
```

Comment:
Above you have a ```RoundOutcome.Result``` object which you named ```result```. Just beneath you have a ```String``` named ```roundResult```. You see the issue here? Semantically, what's truly the result?

I think ```String roundResult``` is in fact your ```outcomeMessage```.